### ISSUE TITLE
autokeras freezing

### ISSUE THREAD

**Issue by HunbeomBak** [keras-team/autokeras#904]
Autokeras is freezing on window10, anaconda python3.6

### Bug Description
first freezing

(autokeras) C:\Users\h2hb0>python test.py
2020-01-17 21:02:07.779028: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-01-17 21:02:11.909961: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-01-17 21:02:11.945781: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.8095
pciBusID: 0000:0a:00.0
2020-01-17 21:02:11.951727: I tensorflow/stream_executor/platform/default/dlopen_checker_stub.cc:25] GPU libraries are statically linked, skip dlopen check.
2020-01-17 21:02:11.956891: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-01-17 21:02:11.960503: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-01-17 21:02:11.969023: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.8095
pciBusID: 0000:0a:00.0
2020-01-17 21:02:11.975248: I tensorflow/stream_executor/platform/default/dlopen_checker_stub.cc:25] GPU libraries are statically linked, skip dlopen check.
2020-01-17 21:02:11.980074: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-01-17 21:02:12.625522: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-17 21:02:12.630207: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0
2020-01-17 21:02:12.632846: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N
2020-01-17 21:02:12.636524: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6354 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080, pci bus id: 0000:0a:00.0, compute capability: 6.1)
Epoch 1/1000
2020-01-17 21:02:31.816334: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-01-17 21:02:32.064241: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-01-17 21:02:32.911773: W tensorflow/stream_executor/cuda/redzone_allocator.cc:312] Internal: Invoking ptxas not supported on Windows
Relying on driver to perform ptx compilation. This message will be only logged once.
    191/Unknown - 9s 46ms/step - loss: 0.5074 - accuracy: 0.8452

then, stop.

second freezing

(autokeras) C:\Users\h2hb0>python test.py
2020-01-17 21:10:17.863189: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_100.dll
2020-01-17 21:10:22.204840: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-01-17 21:10:22.244479: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.8095
pciBusID: 0000:0a:00.0
2020-01-17 21:10:22.251414: I tensorflow/stream_executor/platform/default/dlopen_checker_stub.cc:25] GPU libraries are statically linked, skip dlopen check.
2020-01-17 21:10:22.258515: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-01-17 21:10:22.262048: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
2020-01-17 21:10:22.273303: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties:
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.8095
pciBusID: 0000:0a:00.0
2020-01-17 21:10:22.284777: I tensorflow/stream_executor/platform/default/dlopen_checker_stub.cc:25] GPU libraries are statically linked, skip dlopen check.
2020-01-17 21:10:22.291175: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1746] Adding visible gpu devices: 0
2020-01-17 21:10:22.930518: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-17 21:10:22.934380: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0
2020-01-17 21:10:22.938135: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N
2020-01-17 21:10:22.944225: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1304] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6354 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080, pci bus id: 0000:0a:00.0, compute capability: 6.1)
Epoch 1/1000
2020-01-17 21:10:42.352454: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_100.dll
2020-01-17 21:10:42.588800: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-01-17 21:10:43.403632: W tensorflow/stream_executor/cuda/redzone_allocator.cc:312] Internal: Invoking ptxas not supported on Windows
Relying on driver to perform ptx compilation. This message will be only logged once.
     11/Unknown - 2s 218ms/step - loss: 1.8494 - accuracy: 0.3864

then, stop.

### Reproducing Steps
1. i created a environment for autokeras by using anaconda navigator.
2. i installed tensorflow-gpu (version 2.0.0) by anaconda navigator.
3. i installed autokeras.
4. run bellow code 

from tensorflow.keras.datasets import mnist
import autokeras as ak

(x_train, y_train), (x_test, y_test) = mnist.load_data()

clf = ak.ImageClassifier(max_trials=10)

clf.fit(x_train, y_train)

print('------'*50)
predicted_y = clf.predict(x_test)
print(clf.evaluate(x_test, y_test))

### Setup Details
Include the details about the versions of:
 - OS type and version: window10
 - Python: 3.6
 - autokeras: 1.0.0
 - scikit-learn:
 - numpy:
 - keras:
 - scipy:
 - tensorflow:2.0.0
 - pytorch:

**Comment by haifeng-jin** [comment]
@HunbeomBak Hi, I cannot reproduce the bug on my machine.
Maybe you can try tf 2.1.0?
Let's see if there is anyone else who has this bug.

**Comment by HunbeomBak** [comment]
I checked same code runining on Google colab...

Thank you
